assets/js/pages/ClusterDetails/ClustersList.jsx

```diff
@@ -79,7 +79,7 @@ function ClustersList() {
             const sapSystemData = getSapSystemBySID(allInstances, singleSid);
 
             return [
-              index > 0 && ', ',
+              index > 0 && <br />,
```

Comment:
I wouldn't put a new line, maybe just a space. the newline breaks the table format in my opinion

Reply:
The mockup showed a line break, we still can also have space, what do you think ? 
![image](https://github.com/user-attachments/assets/44c00790-c7cf-4e80-b9a8-ce2dff16d8b7)

For me both works fine :)